package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "algodaily_reverse_string.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "lint": "eslint */*.js"
```

Comment:
This will most likely work, but I am not sure 😄 How did you test it?

Try to familiarize yourself with the concept of _globs_ and notice a specific syntax that [ESLint's docs](https://eslint.org/docs/user-guide/command-line-interface) mention when it comes to globs 🙂